aggregator/handler/type.go

```diff
@@ -33,13 +33,15 @@ const (
 	blackholeType Type = "blackhole"
 	loggingType   Type = "logging"
 	forwardType   Type = "forward"
+	dynamicType   Type = "dynamic"
```

Comment:
As mentioned above, you don't need this type if you change how the flush handler config is structured.